### PR TITLE
Update installation steps

### PR DESCRIPTION
**Pull request by ikus060**
I'm in process of updating the installation documentation in the README to make it clear and easy for every one. While testing the installation I'm facing issue and I'm not sure anybody was aware of them.
Check the FIXME.

Basically, it's it's bad shape. Ubuntu PPA is working, but report wrong version.
@ottok Could you take a look at #305 

Debian PPA is not working
@ottok Is the PPA suposed to work for Debian buster ? I only see packages for sid.

CORP repo on CentOS 7 & 8 is missing librsync v2
@frankcrawford Anything spefici we should do to install librsync v2 ?

### CODE REVIEW

**Comment by frankcrawford** [comment]
@ikus060 For CentOS 7 & 8 you need to enable EPEL as well, as librsync v2 is an EPEL product.

**Comment by frankcrawford** [comment]
I should add, in case it isn't obvious, rdiff-backup will also be an EPEL package in general release, so they are both in the same repo.

**Comment by ikus060** [comment]
@frankcrawford installing `epel-release` make everything working again on CentOS7 &8
Perfect.

@ottok Ubuntu PPA is still reporting "DEV" for the version. I think you already know about this. But just wanted to confirm.

Plz review and merge if everything is OK for you.

File: README.md [hunk 1]
```diff
@@ -39,54 +39,74 @@ provided the file system(s) also support these features.
...
+	sudo python3 get-pip.py
+	sudo pip3 install rdiff-backup
+
+### Other Linux (From pypi)
 
 You need to make sure that the following requirements are met:
 
 * Python 3.5 or higher
-* librsync 1.0.0 or higher
+* librsync 2.0.0
```
[the remark this answers is not in the document]
Comment (ikus060) [reply README.md]: fixed

File: README.md [hunk 2]
```diff
@@ -39,54 +39,74 @@ provided the file system(s) also support these features.
...
+Error: Package: rdiff-backup-2.0.0-1.el7.x86_64 (copr:copr.fedorainfracloud.org:frankcrawford:rdiff-backup)
+           Requires: librsync.so.2()(64bit)
+
+### CentOS 8 (From CORP)
+
+	yum install dnf-plugins-core
+	dnf copr enable frankcrawford/rdiff-backup
+	yum install rdiff-backup
+
+FIXME This is broken
```
[the remark this answers is not in the document]
Comment (ikus060) [reply README.md]: Already fixed in the latest commit

File: README.md [hunk 3]
```diff
@@ -39,54 +39,74 @@ provided the file system(s) also support these features.
...
+	sudo add-apt-repository ppa:rdiff-backup/rdiff-backup-development
+	sudo apt update
+	sudo apt install rdiff-backup
 
-### From our own packaging
+FIXME version reported  is `DEV`.
 
-If you want or need a more recent version than provided by your distribution,
-the [rdiff-project releases its' own packages](https://github.com/rdiff-backup/rdiff-backup/releases), which you can install as follows.
+### CentOS 7 (From CORP)
```
[the remark this answers is not in the document]
Comment (ikus060) [reply README.md]: fixed

**Comment by ericzolf** [comment]
Now that the issue with Travis is fixed, and once #285 is merged, you should rebase/merge master and force again a change to the PR to be redirected to travis-ci.com.

**Comment by ericzolf** [comment]
Again prefix FIX|CHG|NEW...

**Pull request merged**